pytorch/inference/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,204 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu20.04
+
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.10
+
+# PyTorch Binaries
+ARG PT_INFERENCE_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/r1.9.0_inference/20210617-030314/46d3d4fb3c4208f6fcdf1e6aca78ed1fc9155236/gpu/torch-1.9.0-cp38-cp38-manylinux1_x86_64.whl
+ARG PT_TORCHVISION_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/torchvision_build/0.10.0/gpu/torchvision-0.10.0-cp38-cp38-linux_x86_64.whl
+ARG PT_TORCHAUDIO_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/torchaudio_build/0.9.0/gpu/torchaudio-0.9.0-cp38-cp38-linux_x86_64.whl
+
+ARG OPEN_MPI_VERSION=4.0.1
+ARG EFA_PATH=/opt/amazon/efa
```

Comment:
`EFA_PATH` supposed to be used somewhere?

Reply:
Thanks for the feedback, removed EFA_PATH.